docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -67,35 +67,49 @@
     display: flex;
 
     &:not(:last-child) {
-      margin-bottom: hb-calculate-rems(20px);
+      margin-bottom: hb-calculate-rems(10px);
     }
-  }
 
-  &__subcontent-item {
-    display: flex;
-    color: darken($su-color-driftwood, 45%);
-    font-size: hb-calculate-rems(14px);
-
-    @include grid-media-min('md') {
-      font-size: hb-calculate-rems(16px);
-    }
+    // The date-stacked-vertical-card uses the views field to add a pipe between
+    // multiple time fields. Horizontal and Vertical cards do the same thing but
+    // use the .hb-card__subcontent-item class.
+    &-item,
+    .views-field {
+      display: flex;
+      color: darken($su-color-driftwood, 45%);
+      font-size: hb-calculate-rems(14px);
 
-    &::after {
-      content: '';
-      margin: 0 hb-calculate-rems(8px);
-      border-left: 1px solid darken($su-color-driftwood, 45%);
-    }
+      @include grid-media-min('md') {
+        font-size: hb-calculate-rems(16px);
+      }
 
-    &:last-of-type {
       &::after {
         content: '';
-        margin: 0;
-        border: 0;
+        margin: 0 hb-calculate-rems(8px);
+        border-left: 1px solid darken($su-color-driftwood, 45%);
```

Comment:
can we add a comment that this is the pipe CSS

Reply:
It also might be better to use border-right starting on the second child every time (see the bug I commented on with the screenshot above)

Reply:
The border is only applied to the pseudo element which has no width, not the field itself. We remove the pipe from the last-child in the next line. If you're seeing an pipe when you shouldn't be it means there is an empty field in the DOM.